### PR TITLE
deposit -> maintain

### PR DESCRIPTION
**Pull request by myhndl**

### CODE REVIEW

**Comment by myhndl** [comment]
Wait jk missing args, one sec

**Comment by myhndl** [comment]
Okay worth a look now @Rizato

File: {{ cookiecutter.project_slug }}/docker/marketplace.yml [hunk 1]
```diff
@@ -9,8 +11,27 @@ services:
       - marketplace.env
     command:
       - "balancemanager"
-      - "deposit"
-      - {% if cookiecutter.participant_type == "microengine" or cookiecutter.participant_type == "ambassador" %}"100000"{% endif %}{% if cookiecutter.participant_type == "arbiter" %}"10000000"{% endif %}
+      - "maintain"
```
Comment (Rizato) [inline {{ cookiecutter.project_slug }}/docker/marketplace.yml]: I think it makes sense to stay on deposit for arbiters. They don’t want to double send which will happen when they stake.

File: {{ cookiecutter.project_slug }}/docker/marketplace.yml [hunk 2]
```diff
@@ -9,8 +11,27 @@ services:
...
+    image: "polyswarm/polyswarm-client:latest"
+    environment:
+      - PYTHONUNBUFFERED=1
+    env_file:
+      - marketplace.env
+    command:
+      - "balancemanager"
+      - "maintain"
+      - "10000"
+      - "1000"
```
Comment (Rizato) [inline {{ cookiecutter.project_slug }}/docker/marketplace.yml]: I think this will cause 10 deposits when they start

Comment (myhndl) [reply {{ cookiecutter.project_slug }}/docker/marketplace.yml]: Changed

**Pull request merged**